cluster/install.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+####################################################################
+# Base setup for a cluster with 2 nodes hosting the application pods
+####################################################################
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+#
+# Create a KIND cluster
+#
+kind delete cluster --name=curity 2>/dev/null
```

Comment:
I suggest giving the cluster the same name as the repository/example. In that way, it becomes obvious what the cluster is about/comes from if someone forgets to clean up or just inspects the resources after running the scripts.